### PR TITLE
Re-decode events for release

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
Failing test is a remote error

```
[api.etherscan.io/api?module=proxy&action=eth_getTransactionReceipt&txhash=0x3f313e90ed07044fdbb1016ff7986fd26adaeb05e8e9d3252ae0a8318cb8100d&apikey=8JT7WQBB2VQP5C3416Y8X3S8GBA3CVZKP4](https://api.etherscan.io/api?module=proxy&action=eth_getTransactionReceipt&txhash=0x3f313e90ed07044fdbb1016ff7986fd26adaeb05e8e9d3252ae0a8318cb8100d&apikey=8JT7WQBB2VQP5C3416Y8X3S8GBA3CVZKP4) failed with HTTP status code 502 and text <html>
```

File: rotkehlchen/db/upgrades/v42_v43.py [hunk 1]
```diff
@@ -59,6 +64,32 @@ def _remove_old_csv_files(data_dir: Path) -> None:
             log.debug(f'Deleted {csv_file_path} as part of the db upgrade')
 
 
+@enter_exit_debug_log()
+def _reset_decoded_events(write_cursor: 'DBCursor') -> None:
+    """Reset all decoded evm events except the customized ones."""
```
Comment (LefterisJP) [inline rotkehlchen/db/upgrades/v42_v43.py:69]:
```suggestion
    """Reset all decoded evm events except for the customized ones and those in zksync lite."""
```

File: rotkehlchen/db/upgrades/v42_v43.py [hunk 2]
```diff
@@ -67,8 +98,10 @@ def upgrade_v42_to_v43(db: 'DBHandler', progress_handler: 'DBUpgradeProgressHand
     - change hop protocol counterparty value
     - add new supported location: HTX
     - remove coinbasepro credentials
+    - remove CSVs files since they will be replaced by parquet files
+    - remove evm events except for zksync lite
```
Comment (LefterisJP) [inline rotkehlchen/db/upgrades/v42_v43.py:102]:
```suggestion
    - reset decoded evm events except for zksync lite and custom ones
```

File: rotkehlchen/tests/db/test_db_upgrades.py [hunk 3]
```diff
@@ -2370,10 +2370,15 @@ def test_upgrade_db_42_to_43(user_data_dir, messages_aggregator, data_dir):
         msg_aggregator=messages_aggregator,
         resume_from_backup=False,
     )
+    test_addrs = '0xc37b40ABdB939635068d3c5f13E7faF686F03B65'
```
Comment (LefterisJP) [inline rotkehlchen/tests/db/test_db_upgrades.py:2373]:
```suggestion
    test_address = '0xc37b40ABdB939635068d3c5f13E7faF686F03B65'
```

File: rotkehlchen/tests/db/test_db_upgrades.py [hunk 4]
```diff
@@ -2370,10 +2370,15 @@ def test_upgrade_db_42_to_43(user_data_dir, messages_aggregator, data_dir):
...
     with db_v42.conn.read_ctx() as cursor:
         nfts = cursor.execute('SELECT name FROM nfts').fetchall()
         assert nfts == [('yabir.eth',)]
-        # check hop-protocol counterparty is there. If redecoding in upgrade will need to customize
+        # check hop-protocol counterparty is there
+        assert cursor.execute(
+            'SELECT location from history_events WHERE location_label=?',
+            (test_addrs,),
+        ).fetchall() == [(Location.ZKSYNC_LITE.serialize_for_db(),), (Location.POLYGON_POS.serialize_for_db(),)]  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/tests/db/test_db_upgrades.py:2381]: Why did you edit the comment and leave it on top of the new and unrelated code?

```suggestion
        assert cursor.execute(
            'SELECT location from history_events WHERE location_label=?',
            (test_addrs,),
        ).fetchall() == [(Location.ZKSYNC_LITE.serialize_for_db(),), (Location.POLYGON_POS.serialize_for_db(),)]  # noqa: E501
         # check hop-protocol counterparty is there
```

**Review by LefterisJP — APPROVED** [review]
LGTM. Failign test `test_find_yearn_vaults_v2_price` seems like a remtoe error. Should be vcred.
